pkg/internal/export/attributes/attr_defs.go

```diff
@@ -203,6 +203,12 @@ func getDefinitions(groups AttrGroups) map[Section]AttrReportGroup {
 				attr.DBOperation: true,
 			},
 		},
+		RedisClientDuration.Section: {
+			SubGroups: []*AttrReportGroup{&appAttributes, &appKubeAttributes},
+			Attributes: map[attr.Name]Default{
+				attr.DBOperation: true,
```

Comment:
qq: is this what is defined [here](https://github.com/grafana/beyla/pull/891/files#diff-bb430d22224199ef3b4a7c8ae87beea2154a4024bc5c8c1f8c9631dfde491908R69) and that's why `db.statement` is hidden? Why we document hidden and what it means?

Reply:
It means that we don't include db.statement by default in metrics, because it will definitely cause high cardinality. We don't include it in traces by default too, because it might contain sensitive information, but if customers want it they can add it.